src/phases/query/IDHack.yml

```diff
@@ -0,0 +1,39 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  Defines common configurations for IDHack query workloads.
```

Comment:
Is the intention that in the future we will add configurations for the other types of Ids here as well? Maybe it would be good to update the comment to specify this is currently only for Object Ids

Reply:
Great question - I left it open-ended since we could certainly refactor more shared configuration. @cswanson310 is that okay with you?